app/assets/javascripts/account/token_copy.js

```diff
@@ -0,0 +1,15 @@
+$(document).ready(function () {
+  var inputEl = $("#deviceToken");
```

Comment:
jquery objects should be defined as `$input` or `$button` to make it clear of the type